Tensile/Source/TensileCreateLibrary.cmake

```diff
@@ -37,12 +37,12 @@ function(TensileCreateLibraryCmake
   message(STATUS "Tensile_COMPILER            from TensileCreateLibraryCmake : ${Tensile_COMPILER}")
 
   # Tensile_ROOT can be specified instead of using the installed path.
-  set(oneValueArgs Tensile_ROOT)
+  set(oneValueArgs Tensile_BIN)
   cmake_parse_arguments(PARSE "" "${oneValueArgs}" "" ${ARGN})
 
-  if(PARSE_Tensile_ROOT)
+  if(PARSE_Tensile_BIN)
     # python not pre-installed, use scripts downloaded to extern/Tensile
-    set(Tensile_CREATE_COMMAND "${Tensile_ROOT}/TensileCreateLibrary")
```

Comment:
@pfultz2 This line is wrong, it should be PARSE_Tensile_ROOT? Otherwise it does not do what is intended.

Reply:
Yes, but it should be `set(Tensile_CREATE_COMMAND "${Tensile_ROOT}/bin/TensileCreateLibrary")`, because `Tensile_ROOT` gets set to the top-level installed directory automatically [here](https://github.com/ROCmSoftwarePlatform/Tensile/blob/develop/Tensile/cmake/TensileConfig.cmake#L29). So this flag can be removed completely.

Reply:
I agree with this, fixing to get it working this way.